setup.py

```diff
@@ -48,7 +51,7 @@ def get_version():
                 'Programming Language :: Python :: 3',
                 'Programming Language :: Python :: 3.6',
                 'Programming Language :: Python :: 3.7',
-                'Development Status :: 5 - Production/Stable',
```

Comment:
@eduardo-rodrigues thanks for spotting that, it was not intentional but just a copy paste from `particle` x).